Java/src/main/java/com/gildedrose/item/StandardItem.java

```diff
@@ -0,0 +1,21 @@
+package com.gildedrose.item;
+
+public class StandardItem implements CustomisedItem {
+
+    public Item item;
+
+    public StandardItem(Item item) {
+        this.item = item;
+    }
+
+    public void updateState() {
+        item.sellIn -= 1;
```

Comment:
Might be worth using constants for some of these 'magic numbers' as it is not obvious what the numbers 1, 0 mean.